modules/extensions-mule-support/src/test/java/org/mule/runtime/module/extension/mule/internal/validation/InvalidOperationInvocationTestCase.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.extension.mule.internal.validation;
+
+import static org.mule.test.allure.AllureConstants.ReuseFeature.REUSE;
+import static org.mule.test.allure.AllureConstants.ReuseFeature.ReuseStory.OPERATIONS;
+
+import org.mule.functional.junit4.MuleArtifactFunctionalTestCase;
+import org.mule.runtime.core.api.MuleContext;
+import org.mule.runtime.core.api.config.ConfigurationException;
+
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+@Feature(REUSE)
+@Story(OPERATIONS)
+public class InvalidOperationInvocationTestCase extends MuleArtifactFunctionalTestCase {
```

Comment:
rename to something that refers to exclusive optionals. You won't be able to test additional invalid operations in the same class

Reply:
renamed to `InvalidExclusiveOptionalsOperationInvocationTestCase`.